### PR TITLE
fix: shortened the app names and group frameworks in sidebar

### PR DESCRIPTION
**Pull request by Shashwat79802**
# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: versioned_docs/version-2.0.0/quickstart/python-fastapi-twilio.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 id: samples-fastapi-twilio
 title: Sample SMS Sending App (FastAPI-Twilio)
+sidebar_label: FastAPI + Twilio
```
Comment (nehagup) [inline versioned_docs/version-2.0.0/quickstart/python-fastapi-twilio.md:4]: I think we add http in brackets

Comment (Shashwat79802) [reply versioned_docs/version-2.0.0/quickstart/python-fastapi-twilio.md]: Done!!

**Review by nehagup — APPROVED** [review]

**Review by nehagup — COMMENTED** [review]
Screenshot 2023-11-21 at 7 16 05 PM


The Java app is empty! It's better remove that for now if you don't have the sample ready. 

Rest looks good! 👏

**Comment by Shashwat79802** [comment]
> Screenshot 2023-11-21 at 7 16 05 PM
> The Java app is empty! It's better remove that for now if you don't have the sample ready.
> 
> Rest looks good! 👏

Removed!!

**Pull request merged**